README.md

```diff
@@ -1,9 +1,10 @@
 
 # gocd-cypress
 
-Cypress runner for GoCD that:
+This package supports running Cypress in GoCD by offering the following features:
 
-- runs test in docker container when a CI environment is detected
+- start the local development server of your application
```

Comment:
This is an optional feature and not the most important one. I would rephrase the sentence for optionality, and make it be the second bullet point. (For example, typically we don't need a local dev server when running end-to-end tests.)

Reply:
Good point, I will reorder it.